centos_ci.sh

```diff
@@ -95,9 +95,9 @@ function perform_artifacts_upload() {
   if [[ "$GIT_BRANCH" = "origin/master" ]]; then
     # http://stackoverflow.com/a/22908437/1120530; Using --relative as --rsync-path not working
     mkdir -p crc/master/$BUILD_NUMBER/
-    cp out/* crc/master/$BUILD_NUMBER/
+    cp -R out/* crc/master/$BUILD_NUMBER/
     RSYNC_PASSWORD=$1 rsync -a --relative crc/master/$BUILD_NUMBER/ minishift@artifacts.ci.centos.org::minishift/crc/
-    echo "Find Artifacts here http://artifacts.ci.centos.org/${REPO_OWNER}/${REPO_NAME}/master/$BUILD_NUMBER ."
+    echo "Find Artifacts here http://artifacts.ci.centos.org/minishift/crc/master/$BUILD_NUMBER ."
```

Comment:
Is this `minishift` change intended?

Reply:
We are using the minishift namespace as we couldn't get the crc one.

Reply:
What happened to $REPO_OWNER and #REPO_NAME ?

Reply:
@cfergeau @gbraad This is c&p mistake from the minishift ci script. these env variables are not present in the CI, in minishift they were defined in the script.

$REPO_OWNER here: https://github.com/minishift/minishift/blob/0381375335880a9f825a7a30266b03d493cd221c/centos_ci.sh#L24
and $REPO we get from the ci environment itself, https://github.com/minishift/minishift/blob/0381375335880a9f825a7a30266b03d493cd221c/centos_ci.sh#L32